src/crawlers/cheerio_crawler.js

```diff
@@ -61,7 +67,7 @@ const DEFAULT_OPTIONS = {
  *
  * The crawler finishes when there are no more {@link Request} objects to crawl.
  *
- * By default, `CheerioCrawler` downloads HTML using the
+ * By default, `CheerioCrawler` downloads HTML or XML using the
```

Comment:
The last PR comment says that it doesn't work with XML by default. There are a few places in the docs where this needs to be changed back.